modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/logging/CorrelationLoggingTest.java

```diff
@@ -167,29 +173,29 @@ public void testRetrieveDefaultCorrelationLoggingConfigsTest() throws Exception
     dependsOnMethods = { "testRetrieveDefaultCorrelationLoggingConfigsTest" })
     public void testEnableAllCorrelationLoggingConfigsTest() throws Exception {
 
-        configureCorrelationLoggingComponent(new String[] {"http", "jdbc", "synapse", "ldap", "method-calls"}, true);
+        configureCorrelationLoggingComponent(new String[] {"http", "synapse", "ldap", "method-calls"}, true);
```

Comment:
Comment why 'jdbc' is skipped. (skipping this is fine anyway because these are tested separately in testSpecificCorrelationLoggingConfigsTest too)

Reply:
Added in https://github.com/wso2/product-apim/pull/13088/commits/2c7b40a5f8ca5cab63ad8b5cb0d8bc511237cc5a